libpdbg/sbe_api.c

```diff
@@ -222,16 +222,31 @@ int sbe_ffdc_get(struct pdbg_target *target, uint32_t *status, uint8_t **ffdc, u
 	return 0;
 }
 
+// Get fsi target from proc type parent target.
+static struct pdbg_target* sbe_get_fsi_target(struct pdbg_target *proc)
+{
+	char path[16];
+	sprintf(path, "/proc%d/fsi", pdbg_target_index(proc));
```

Comment:
What if the path is longer than 16 characters?

Reply:
pdbg_target_index is uint32. 16 seems to be good ,

Reply:
If pdbg_target_index is large, then your string will be 20 bytes long.